alchemist/alchemist-engine/src/test/java/it/unibo/alchemist/core/tests/TestConcurrency.java

```diff
@@ -117,4 +117,4 @@ protected Object createT() {
         }
     }
 
-}
```

Comment:
Get this file from the mainline

Reply:
Done

Reply:
It still pops up in this diff, so it's not done :)